README.md

```diff
@@ -76,3 +76,30 @@ Note that production usage should always create the user with a password. For ex
 
 Automated test do not enforce that a password is set because many development environments
 (OSX, Circle CI) configure `pg_hba.conf` for trusted login from localhost.
+
+## Migration guide: 1.x => 2.x
+
+### Encryption
+
+If you are updating from microcosm-postgres 1.x to 2.x and use encryption, there are a couple of
+configuration updates you will need to add in order to keep encryption working as expected.
+
+* the `multi_tenant_key_registry` has two new parameters to account for `aws-encryption-sdk` updates
+* `account_ids` should be set to the list of AWS account IDs that contain the keys being using for encryption
+* `partitions` should be the list of AWS partitions used to access your encryption keys. This should be a single
+item per registry entry
+
+General configuration will look like the existing configuration used for the registry (e.g. using `;` as a delimiter
+between items per registry entry).
+
+Full configuration may thus look something like this:
+
+    multi_tenant_key_registry=dict(
+         context_keys="client-id1,client-id2",
+         key_ids="key1a;key1b,key2a,key2b",
```

Comment:
I think you mean `key1a;key1b,key2a;key2b` here?

Reply:
Yes, good eye.